aws/provider.go

```diff
@@ -278,6 +278,8 @@ func Provider() terraform.ResourceProvider {
 			"aws_secretsmanager_secret_version":             dataSourceAwsSecretsManagerSecretVersion(),
 			"aws_servicequotas_service":                     dataSourceAwsServiceQuotasService(),
 			"aws_servicequotas_service_quota":               dataSourceAwsServiceQuotasServiceQuota(),
+			"aws_sfn_activity":                              dataSourceAwsSfnActivity(),
+			"aws_sfn_state_machine":                         dataSourceAwsSfnStateMachine(),
```

Comment:
Please note that there is an earlier submission for the `aws_sfn_state_machine` data source that has recently received a review: https://github.com/terraform-providers/terraform-provider-aws/pull/10932